iphone/cli/hooks/storyboard.js

```diff
@@ -0,0 +1,591 @@
+/**
+ * iOS build hook that scans for available storyboards from modules
+ * and then configures them in the Xcode project
+ */
+
+// TO DO: Can we merge storyboard.js and farmework.js
+
+'use strict';
+
+const fs = require('fs-extra');
+const IncrementalFileTask = require('appc-tasks').IncrementalFileTask;
+const path = require('path');
+
+exports.cliVersion = '>=3.2.1';
+
+exports.init = function (logger, config, cli) {
+	let storyboardManager = new StoryboardManager(cli, config, logger);
```

Comment:
why use `let` here? it doesn't get re-assigned/changed, so could be `const` (or not even assigned to a variable)

Reply:
Also, why re-order the passed in arguments (`logger, config, cli -> cli, config, logger`)? Ideally we try and retain arg order just purely for developer sanity sake (not having to look up the order for every call)

Reply:
Done.